README.md

```diff
@@ -146,13 +146,15 @@ you may need to add to your path. Then, you can call `pact` as-is.
 
 The fastest way to build and run Pact is to use the Nix package manager
 which has binary caching capabilities that allow you to download pre-built
-binaries for everything needed by Pact. For detailed instructions see [our
-wiki](https://github.com/kadena-io/pact/wiki/Building-Kadena-Projects).
-
+binaries for everything needed by Pact.
+We use nix flakes to build pact and its dependencies. To build the Pact executable:
+```bash
+nix build
+```
```

Comment:
I have 2 comments about this:
1) We should probably mention that people new to Nix will likely need to add `--experimental-features "nix-command flakes"` to this line.
2) We're removing the link to the [Building With Nix](https://github.com/kadena-io/pact/wiki/Building-Kadena-Projects) page, which contains the instructions for adding our binary cache.

Maybe it would be best to leave that link intact and update the wiki page instead since we've moved to Nix flakes in almost all of our repositories.

Reply:
I've actually been wondering whether we should have a separate repo that exposes all of our projects as Nix artifacts, i.e. a single entry point. Such a repo would also be the best place for this sort of documentation. So maybe it would be best to just briefly mention the chainweb binary cache and its public key here and move on. Once we have the Kadena Nix repo, we can add a "For detailed instructions, check out the Kadena Nix repo" line to the Readme.

Reply:
I'm in favor of mentioning chainweb cache and once we have a Kadena nix repo (which I like as a starting point), revisit the docs.

Reply:
I just noticed `chainweb-node` refers to the pact wiki. We should update the wikis section.